doc/en/components/charts/chart-features-navigation.md

```diff
@@ -0,0 +1,216 @@
+
+## Data Chart UI Navigation Overview
+
+In order to allow navigation in the UI, you need to set either the `IsHorizontalZoomEnabled` and/or the `IsVerticalZoomEnabled` properties of the chart to true, depending on the direction that you wish to allow zooming. This will allow you to zoom the data chart by rolling a mouse wheel over it.
+
+It is also possible to zoom or pan simply by clicking the buttons on the mouse or using touch. The `DefaultInteraction` property of the data chart determines what happens on mouse click or touch events. This property defaults to `DragZoom` and when set to this with zooming enabled, clicking and dragging will place a preview rectangle over the plot area that will become the zoomed area of the chart. This `DefaultInteraction` property can also be set to either `DragPan` to allow panning or `None` to prevent these operations.
+
+The following code snippet demonstrates how you can enable base UI navigation in the $PlatformShort$ data chart component:
+
+```razor
+<DataChart Height="400px" Width="100%"
+    IsHorizontalZoomEnabled="true"
+    IsVerticalZoomEnabled="true">
+</DataChart>
+```
+
+```html
+<igx-data-chart #chart
+    [dataSource]="data"
+    width="100%"
+    height="400px"
+    isHorizontalZoomEnabled="true"
+    isVerticalZoomEnabled="true">
+</igx-data-chart>
+```
+
+```tsx
+<IgrDataChart dataSource={this.data}
+    width="100%"
+    height="400px"
+    isHorizontalZoomEnabled="true"
+    isVerticalZoomEnabled={true} />
+```
+
+```html
+<igc-data-chart id="chart"
+    width="100%"
+    height="400px"
+    is-horizontal-zoom-enabled="true"
+    is-vertical-zoom-enabled="true">
+</igc-data-chart>
+```
+
+## Data Chart Navigation with Mouse and Keyboard
+
+Navigation in the $PlatformShort$ data chart component can happen with either the mouse or the keyboard once enabled. The following operations can be invoked using the following mouse or keyboard operations by default:
+
+- **Panning**: Using arrow keys on keyboard or holding the Shift key and clicking and dragging with the mouse.
+- **Zoom In**: Using the PageUp key on keyboard or rolling the mouse wheel up.
+- **Zoom Out**: Using the PageDown key on keyboard or rolling the mouse wheel down.
+- **Fit to Chart Plot Area**: Home key on the keyboard. There is no mouse operation for this.
+- **Area Zoom**: Click and drag mouse within the plot area with the `DefaultInteraction` property set to its default, `DragZoom`.
+
+The zoom and pan operations can also be enabled by using modifier keys by setting the `DragModifier` and `PanModifier` properties, respectively. These properties can be set to the following modifier keys, and when pressed, the corresponding operation will be enabled:
+
+- Shift
+- Alt
+- Control
+- Windows Key
+- Apple Key
+- None
+
+The following code snippet demonstrates how you can enable UI navigation in the data chart. The following will only allow zooming while holding the **Shift** key and panning while holding the **Alt** key:
+
```

Comment:
add a note that says the following code snippets also apply to CategoryChart and FinancialChart

Reply:
pending